binding/web/src/dialog.ts

```diff
@@ -244,5 +272,9 @@ export const DIALOGS: { [key: string]: typeof Dialog | { [key: string]: typeof D
     "default": Phi2QADialog,
     "qa": Phi2QADialog,
     "chat": Phi2ChatDialog
-  }
+  },
+  "phi3": {
+    "default": Phi3ChatDialog,
+    "chat": Phi3ChatDialog,
+  },
```

Comment:
phi3 can just lead to `Phi3ChatDialog` directly since there aren't different varieties